cmd/podman/images/pull.go

```diff
@@ -154,15 +154,22 @@ func imagePull(cmd *cobra.Command, args []string) error {
 	}
 	// Let's do all the remaining Yoga in the API to prevent us from
 	// scattering logic across (too) many parts of the code.
-	pullReport, err := registry.ImageEngine().Pull(registry.GetContext(), args[0], pullOptions.ImagePullOptions)
-	if err != nil {
-		return err
+	pulledImages := make([]string, 0)
+	for _, arg := range args {
+		pullReport, err := registry.ImageEngine().Pull(registry.GetContext(), arg, pullOptions.ImagePullOptions)
+		if err != nil {
+			return err
```

Comment:
When we try to do multiple operations at once, we complete the tasks, and print the errors.  Look how podman stop works.
If I say

podman pull alpine bogus fedora ubi

I would want alpine fedora and ubi to be pulled, and then the command fail with bogus does not exist.

Reply:
I think that is possible because the ContainerStop function accepts an array of strings and here it's just a single string. And passing an array makes it possible to get an accumulated response but in this case we would have to change the code for Pull in various places. 
But if you think that this is possible, then this PR is editable by maintainers. So please make the change. Thanks!

Reply:
No you just need something like:
```
var errs utils.OutputErrors
for _, arg := range args {
		pullReport, err := registry.ImageEngine().Pull(registry.GetContext(), arg, pullOptions.ImagePullOptions)
		if err != nil {
                        errs = append(errs, err)
			continue
              }
    ...
    }
    return errs.PrintErrors()
}

Reply:
I have updated the code, but why arent the checks running?

Reply:
Yes, it looks like a `master` -> `main` issue. Hm ... maybe try closing and re-opening. Then repush?

Reply:
@vrothberg I fixed that, but some tests are failing. Am I missing something?

Reply:
Tests are currently running.  I'll wait for them to finish and will have a look later.

Reply:
Looks like it is unable to resolve the tag I am using for the alpine image
[Error: Error initializing source docker://alpine:edge: Error reading manifest edge in quay.io/libpod/alpine: manifest unknown: manifest unknown](https://storage.googleapis.com/cirrus-ci-6707778565701632-fcae48/artifacts/containers/podman/4715965364568064/html/int-podman-fedora-34-rootless-host.log.html)
But this works just fine locally for me?